src/Controls/src/Core/Platform/Windows/Extensions/ToolbarExtensions.cs

```diff
@@ -84,5 +84,10 @@ public static void UpdateToolbarDynamicOverflowEnabled(this MauiToolbar platform
 
 			platformToolbar.CommandBar.IsDynamicOverflowEnabled = toolbar.DynamicOverflowEnabled;
 		}
+
+		private static void UpdateBackButtonVisibility(MauiToolbar platformToolbar, Toolbar toolbar)
+		{
+			platformToolbar.IsBackButtonVisible = toolbar.BackButtonVisible ? NavigationViewBackButtonVisible.Visible : NavigationViewBackButtonVisible.Collapsed;
```

Comment:
Can apply a small change to improve the readability by formatting the ternary operation across multiple lines:

```
platformToolbar.IsBackButtonVisible = 
        toolbar.BackButtonVisible 
            ? NavigationViewBackButtonVisible.Visible 
            : NavigationViewBackButtonVisible.Collapsed;
```

Reply:
@jsuarezruiz I have updated the codes based on your feedback. Could you please review and let me know if you have any further concerns?